tools/Google.Cloud.Tools.ReleaseManager/SuggestSmokeTestsCommand.cs

```diff
@@ -0,0 +1,164 @@
+﻿// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using Google.Cloud.Tools.Common;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Converters;
+using Newtonsoft.Json.Serialization;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using System.Threading.Tasks;
+
+namespace Google.Cloud.Tools.ReleaseManager
+{
+    /// <summary>
+    /// Command that looks at all the RPCs in a client library and considers which ones might be suitable for smoke tests.
+    /// (This is currently very simplistic, but can become more sophisticated over time.)
+    /// The smoke tests are not written to disk automatically, as they should definitely be checked by hand. For example,
+    /// the Vision API's ProductSearchClient has a PurgeProducts parameterless method - we can't easily tell that that's
+    /// dangerous whereas other parameterless RPCs are safe. (A really sophisticated version of this tool might examine
```

Comment:
You have an amusingly low bar for "really sophisticated" :)

Reply:
Maybe "complicated" in this case - we'd have to scan the protos to find the right RPC, then parse the HTTP bindings, which we don't currently use.
A less complex but really, really ugly approach would be to construct the default settings for the RPC and see if it's retryable. Assume we don't want to do anything that can't be retried...